### PR TITLE
#161646092 Add floating create and edit buttons 

### PR DESCRIPTION
**Pull request by pabishai**
#### What does this PR do?
Enable user to create article from the root page which lists all articles
Enable user to edit an article from the single article page

#### Description of Task to be completed?
Add floating button on all articles page to create an article. Only appears if a user is authenticated
Add a floating button on a single article page to edit an article. Only appears if a user is authenticated and the author of the article.

#### How should this be manually tested?
Clone: `https://github.com/andela/mag-ah-frontend.git`
Checkout to: `ch-floating-buttons-161646092 `
Install requirements: `npm install`
Create env file: `mv .envsample .env`
Run app: `npm start`
Login into the application

Navigate to the following url:

`http://localhost:3000/articles` to view the floating button.
`http://localhost:3000/{slug}` where the article is authored by you. You should see an edit button

#### What are the relevant pivotal tracker stories?
[Users should be able to view and edit their profiles](https://www.pivotaltracker.com/story/show/161646092)

#### Screenshots

`Create article`

screen shot 2018-11-01 at 14 35 40

`Update article`

screen shot 2018-11-01 at 14 35 55

### CODE REVIEW

**Comment by Bileonaire** [comment]
Looks good to me. I approve the changes. It works well

File: src/views/App/App.scss [hunk 1]
```diff
@@ -682,4 +678,17 @@ h1 {
...
+  bottom: 50px;
+  right: 50px;
+  width: 50px;
+  height: 50px;
+  font-size: 2rem;
+}
+
+.fa-pencil-alt{
+  font-size: 1.5rem;
 }
```
Comment (kimotho-njoki) [inline src/views/App/App.scss]: What is missing here?

Comment (pabishai) [reply src/views/App/App.scss]: An empty line

Comment (pabishai) [reply src/views/App/App.scss]: I hope this is what you meant

**Pull request merged**